.circleci/config.yml

```diff
@@ -25,13 +25,25 @@ jobs:
             - tests
             - scripts/sphinx/build/html
 
-  vscode:
+  vscode_latest:
     machine:
       image: ubuntu-2004:202201-02
     steps:
       - attach_workspace:
           at: /tmp/workspace
-      - run: docker build -f /tmp/workspace/.circleci/dockerfiles/wasm -t wake-wasm .
+      - run: docker build -f /tmp/workspace/.circleci/dockerfiles/wasm-stable -t wake-wasm .
```

Comment:
Should this be `docker build -f .../wasm-latest`?

Also, this may just be me not knowing CircleCI, but where do we set which of these are required?  Is that a separate UI?

Reply:
> Should this be docker build -f .../wasm-latest?
yep, bad copy+passte

> Also, this may just be me not knowing CircleCI, but where do we set which of these are required? Is that a separate UI?
working on that bit of code atm